canonical-websites/www.ubuntu.com
# Rename BootStack legal section to managed-services

## Done

- Include kubernetes in the BootStack service description
- Created an archive of the current version, but am not showing it currently as there is no material change

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [BootStack service description](https://www.ubuntu.com/legal/bootstack/service-description)
- compare to the [copy doc](https://docs.google.com/document/d/1wiUeISVo9jJ2hVPRiXKnnX__R3eyFfJ5PB15-v9bEAM/edit#heading=h.qx09sxsi9xd3)

## Issue / Card

Fixes #2937
